README.rst

```diff
@@ -406,6 +406,28 @@ information by performing lookups on the OUI
 
 This feature is enabled by default.
 
+``OPENWISP_MONITORING_DEVICE_RESOURCES_THRESHOLDS``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------------------------------------------+
+| **type**:    | ``dict``                                        |
++--------------+-------------------------------------------------+
+| **default**: | ``{'memory': 0.95, 'cpu': 0.90, 'disk': 0.80}`` |
++--------------+-------------------------------------------------+
+
+This setting allows to change the thresholds of device resources metrics namely
+``Disk usage``, ``Memory usage`` and ``CPU usage``.
+For example, if we want to set thresholds for ``Memory usage`` and ``Disk usage`` metrics to
+``0.90`` and ``0.85`` respectively, then you can use the following configuration:
+
+.. code-block:: python
+
+    OPENWISP_MONITORING_DEVICE_RESOURCES_THRESHOLDS = {
+        'memory': 0.90,
+        'cpu': 0.90,
+        'disk': 0.85,
```

Comment:
Please don't. We need a generic solution for this as mentioned in https://github.com/openwisp/openwisp-monitoring/issues/106
Remove this logic please.

Reply:
I forgot that there's a separate issue existing which states it more clearly, thanks for the reminding :smile: